dpnp/dpnp_algo/dpnp_arraycreation.py

```diff
@@ -10,6 +12,7 @@
     "dpnp_geomspace",
     "dpnp_linspace",
     "dpnp_logspace",
+    "nd_grid",

```

Comment:
probably to add dpnp prefix:
```suggestion
    "dpnp_nd_grid",

```